starter/increment-counter.rest

```diff
@@ -0,0 +1 @@
+GET http://localhost:7071/api/DfSpikeOrch_HttpStart
```

Comment:
your current host.json nixes `/api`
if you rename the func this will need to change

Reply:
Nope, I am keeping that